QuizBowlDiscordScoreTracker/BotConfiguration.cs

```diff
@@ -58,6 +58,7 @@ public BotConfiguration()
 #pragma warning restore CA1819 // Properties should not return arrays
 
         public string BotToken { get; set; }
+        public System.Uri webBaseURL { get; set; }
```

Comment:
Consider importing System with a using to keep the type name cleaner

Reply:
Also consider naming it WebBaseURL to make it consistent with other properties (first letter capitlaized)

Reply:
fixed in b5da9fd